Spokestack/TextToSpeech.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  TextToSpeech.swift
+//  Spokestack
+//
+//  Created by Noel Weichbrodt on 11/15/19.
+//  Copyright © 2019 Pylon AI, Inc. All rights reserved.
+//
+
+import Foundation
+
+@objc public enum TTSInputType: Int {
+    case text
+    case ssml
+}
+
+@objc class TextToSpeech: NSObject {
+    private var delegate: TextToSpeechDelegate
+    private var configuration: SpeechConfiguration
+    
+    /// Initializes a new text to speech instance.
+    /// - Parameter delegate: Delegate that receives text to speech events.
+    /// - Parameter configuration: Speech configuration parameters.
+    init(_ delegate: TextToSpeechDelegate, configuration: SpeechConfiguration) {
+        self.delegate = delegate
+        self.configuration = configuration
+        super.init()
```

Comment:
@noelweichbrodt Is there a necessity to have a `strong` reference to the `delegate`? Should that be a `public` scope?

Something like: `weak public var delegate: TextToSpeechDelegate?`

Reply:
good point!